csv_importer/ops.py

```diff
@@ -82,7 +64,7 @@ class CSV_FH_import(bpy.types.FileHandler):
     @classmethod
     def poll_drop(cls, context):
         # Allow drag-and-drop
-        return context.area
+        return context.area and context.area.type == "VIEW_3D"
```

Comment:
```suggestion
        return context.area
```
here, I think it's better when the drag'n'drop area is not restricted to the 3D viewport, but anywhere in the Blender IDE.